compiler/qsc_frontend/src/resolve.rs

```diff
@@ -68,6 +68,10 @@ pub(super) enum Error {
     #[error("`{0}` not found")]
     #[diagnostic(code("Qsc.Resolve.NotFound"))]
     NotFound(String, #[label] Span),
+
+    #[error("This name results in ambiguity in a prelude namespace.")]
```

Comment:
It'd be nice to explicitly mention two of the namespaces where the name is declared, like `Ambiguous` does. It's OK that there are no spans to point to in this case (since the opens are implicit).